xrpl/models/transactions/permissioned_domain_delete.py

```diff
@@ -0,0 +1,21 @@
+"""Model for PermissionedDomainDelete transaction type."""
+
+from dataclasses import dataclass, field
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class PermissionedDomainDelete(Transaction):
```

Comment:
Should add docstring description for the transaction and its fields based on the spec.

Reply:
thanks, fixed in [20762f8](https://github.com/XRPLF/xrpl-py/pull/773/commits/20762f8cbd2f438c28084c2735fbd9db6c8edfcc)